src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
+html,
+body,
+.header,
+.img-link,
+.nav-link,
+.nav-list {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Avoid using the `*` selector to set styles for all elements. It impacts performance. Try to set styles only for elements that require them. For example, instead of setting `box-sizing: border-box;` for all elements, set it only for the elements that need it.